### PR TITLE
Update oudated CutomLoader code example

### PR DESCRIPTION
**Pull request by xinbinhuang**
Simple typo fits on the example doc

### CODE REVIEW

File: src/jinja2/loaders.py [hunk 1]
```diff
@@ -55,7 +55,7 @@ def get_source(self, environment, template):
                 if not exists(path):
                     raise TemplateNotFound(template)
                 mtime = getmtime(path)
-                with file(path) as f:
+                with open(path) as f:
                     source = f.read().decode('utf-8')
```
Comment (davidism) [inline src/jinja2/loaders.py:59]: Decode needs to be removed since read already returns a string, not bytes.

**Comment by davidism** [comment]
It's not a typo, it's just a really, really old example. Needs more updating for Python 3.

**Comment by xinbinhuang** [comment]
> It's not a typo, it's just a really, really old example. Needs more updating for Python 3.

@davidism I see. Thanks for the info. Can I assume that the codebase is all updated to Python 3 (I saw a commit using `pyupgrade`), but some code examples may still be outdated? 

I will make some PRs if I notice more.

Thanks
Bin

**Comment by davidism** [comment]
The code in master is all Python 3 now, but I didn't have the capacity to do all the docs too. https://github.com/pallets/meta/issues/26 is about updating the docs for all projects.

**Pull request merged**